amenities.mss

```diff
@@ -1004,19 +1039,26 @@
     text-dx: 12;
   }
 
+  [feature = 'tourism_information'][zoom >= 19],
+  [feature = 'tourism_information']["information"='office'][zoom >= 17],
   [feature = 'tourism_artwork'][zoom >= 19],
   [feature = 'tourism_gallery'][zoom >= 17],
-  [feature = 'historic_city_gate'][zoom >= 17],
   [feature = 'tourism_museum'][zoom >= 17],
```

Comment:
Why changing the render of these `man_made` / `tourism`?

Reply:
I just reordered the list to have a to have the same kind of POI together, and changed @memorial to @culture because @memorial has no sense here.
Final rendering is the same.